Expanded the introduction part of the reference baseline page

Yemaneberhan-Lemma opened the pull request.

hussainweb commented on content/docs/how-we-work/baseline/_index.md:
```diff
@@ -7,7 +7,13 @@ description: This is our reference baseline of technologies and techniques we us
+
+Additionally, it encourages continuous improvement by providing a foundation for ongoing evaluation and optimization. Ultimately, the reference baseline sets us up for sustained success and growth, addressing current challenges while paving the way for future scalability.
+
+At this time, this baseline is standardized for our support projects but will eventually also apply to our managed projects.
```
Let's use the word CDM instead of support.

hussainweb replied:
```suggestion
At this time, this baseline is standardized for our projects in CDM but will eventually also apply to our managed projects.
```

Kalaiselvan88 commented on content/docs/how-we-work/baseline/_index.md:
```diff
@@ -7,7 +7,13 @@ description: This is our reference baseline of technologies and techniques we us
 # Our Reference Baseline
 
-This is our reference baseline of technologies and techniques we use on any project. At this time, this baseline is standardized for our support projects but will eventually also apply to our managed projects.
+Establishing a reference baseline for our projects is crucial for enhancing the overall efficiency and effectiveness of our operations. This initiative aims to standardize processes, reduce decision fatigue, and improve productivity by ensuring all projects adhere to a consistent set of standards and practices.
```
@Yemaneberhan-Lemma Can we rephrase this into points having below structure?

> Establishing a reference baseline for our projects is crucial for enhancing the overall efficiency and effectiveness of our operations.

followed by 

It benefits the below stakeholders in the following ways.

1. Customer
2. Team
3. Axelerant

I think above calls out the benefits for all stakeholders more clearly and since this is public it enhances the perspective of Axelerant as well that we are considering everyone.

Kalaiselvan88 replied:
An example : https://www.thoughtworks.com/en-in/insights/topic/sensible-defaults

Kalaiselvan88 reviewed (changes requested):
I think the structure can be made more into pointers as mentioned rather than having as paragraphs to bring more clarity to audience

hussainweb commented:
@Kalaiselvan88, based on your comments, what do you think of creating a page called "Why baseline?" that goes deeper into the topic? Let's not have subheadings for this page for the introduction itself when there are already so many subheadings for the baseline items themselves.

Kalaiselvan88 commented:
> @Kalaiselvan88, based on your comments, what do you think of creating a page called "Why baseline?" that goes deeper into the topic? Let's not have subheadings for this page for the introduction itself when there are already so many subheadings for the baseline items themselves.

Yes @hussainweb that makes sense and allows us to explain it in detail

Yemaneberhan-Lemma commented:
@Kalaiselvan88 @hussainweb: Isn't this new introduction adequate to show the benefits without an additional page?

hussainweb commented:
@Yemaneberhan-Lemma, it's too big right now. We need to scroll down to get to the first section. That's too big in my opinion. We should condense this as much as possible and move the elaborate content in a sub-page. cc @Kalaiselvan88

Yemaneberhan-Lemma commented:
I added a new separate page for the detailed benefits of the reference baseline and added link in the introduction part of the reference baseline index page.
@hussainweb @Kalaiselvan88

hussainweb commented on content/docs/how-we-work/baseline/why-ref-baseline.md:
```diff
@@ -0,0 +1,35 @@
+---
+title: Why Reference Baseline?
+weight: 1
```
Shall we try a `weight` of 0? Right now, this page appears second in the list. See https://deploy-preview-204--axelerant-engg-handbook.netlify.app/docs/how-we-work/baseline/.

hussainweb commented on content/docs/how-we-work/baseline/_index.md:
```diff
@@ -2,12 +2,14 @@
 # Our Reference Baseline
 
-This is our reference baseline of technologies and techniques we use on any project. At this time, this baseline is standardized for our support projects but will eventually also apply to our managed projects.
+Establishing a reference baseline enhances operational efficiency. It ensures quality, consistency, and clear communication for customers; reduces decision fatigue, boosts productivity, and fosters collaboration for our team; and supports scalability, compliance, and continuous improvement for Axelerant. This comprehensive approach addresses current challenges and prepares us for future growth. [ ...
```
Let's not use the word "Establishing". It signals that we are still deciding whether to define a baseline or not, which is not true. We can instead say "Adopting the baseline on a project".

hussainweb commented on content/docs/how-we-work/baseline/why-ref-baseline.md:
```diff
@@ -0,0 +1,35 @@
+
+* **Reduced Decision Fatigue**: A reference baseline minimizes the number of decisions team members need to make by providing clear guidelines and standards. This reduction in decision fatigue allows our team to focus more on critical tasks and problem-solving.
+
+* **Enhanced Productivity through Streamlined Workflows and Standardized Tools**: With standardized workflows and tools, our team can work more efficiently and effectively. This consistency reduces the learning curve for new projects and improves overall productivity.
+
+* **Fosters Better Collaboration with a Common Framework**: A common framework promotes better collaboration among team members by providing a shared understanding of processes and standards. This shared understanding reduces misunderstandings and enhances teamwork.
```
Both of these sound very similar to me. Let's replace one of them with "Faster onboarding".

hussainweb commented on content/docs/how-we-work/baseline/why-ref-baseline.md:
```diff
@@ -0,0 +1,35 @@
+
+* **Faster Response Times through Streamlined Processes**: By standardizing processes, we can streamline our workflows, enabling faster response times to customer needs and inquiries. This agility helps us to address issues promptly and deliver solutions more quickly.
+
+* **Clear Communication due to Standardized Practices**: Standardized practices and processes foster clearer communication between our team and customers. Customers receive consistent information and updates, enhancing transparency and trust.
```
Is this really a thing? How does the baseline help in communication (except communicating that we are going to adopt a baseline)?

Yemaneberhan-Lemma replied:
> Is this really a thing? How does the baseline help in communication (except communicating that we are going to adopt a baseline)?

@hussainweb:  When the practices and processes are standardized, the communication will be clearer. Doesn't this make sense?

Yemaneberhan-Lemma commented:
@hussainweb: I made changes as per your today's comments.

hussainweb reviewed (approved):
I am happy with the changes. @Kalaiselvan88, please go through this once.

Kalaiselvan88 commented on content/docs/how-we-work/baseline/_index.md:
```diff
@@ -2,12 +2,14 @@
-This is our reference baseline of technologies and techniques we use on any project. At this time, this baseline is standardized for our support projects but will eventually also apply to our managed projects.
+Adopting the baseline on a project enhances operational efficiency. It ensures quality, consistency, and clear communication for customers; reduces decision fatigue, boosts productivity, and fosters collaboration for our team; and supports scalability, compliance, and continuous improvement for Axelerant. This comprehensive approach addresses current challenges and prepares us for future growth.  ...
+
+At this time, this baseline is standardized for our projects in CDM but will eventually also apply to our managed projects.
```
@Yemaneberhan-Lemma Can we link the CDM and Managed words to our pages

CDM - https://www.axelerant.com/delivery-engagement-models/continuous-development-and-maintenance
Managed - https://www.axelerant.com/delivery-engagement-models/managed-projects

Kalaiselvan88 reviewed (approved):
I am approving this as the content looks good. However I think the open comments are important IMO as they are going to 404 and order is also wrong.
